Remotion/Web/Development.WebTesting/WebDriver/BrowserConfigurationExtensions.cs

```diff
@@ -31,17 +30,6 @@ namespace Remotion.Web.Development.WebTesting.WebDriver
   /// </summary>
   public static class BrowserConfigurationExtensions
   {
-    /// <summary>
-    /// Gets a flag indicating if <paramref name="browserConfiguration"/> represents <b>Internet Explorer</b> by testing if the passed object 
-    /// implements <see cref="IInternetExplorerConfiguration"/>.
-    /// </summary>
-    public static bool IsInternetExplorer ([NotNull] this IBrowserConfiguration browserConfiguration)

```

Comment:
we want to keep those non-public. In case we need them in muliple places, we could do this via obsoleteattribute, too, and just disable the obsolete-check on demand at the usage sites. Plus, add an EditorBrowsable-Attribute with Never to the methods to hide them from other users.

Reply:
Here's a quick overview what I did with the extension methods:

* The AccessibilityTest one I moved to the `BocAccessibilityTest` assembly, no problem there
* `IsInternetExplorer` on `IBrowserConfiguration` was moved to `TestInfrastructure` because it is only used in tests
* `IsInternetExplorer` on `Coypu.Browser` was marked obsolete but left where it was in the production code, since it is actually used in the production code.